### ISSUE TITLE
Plugin doesn't work osTicket 1.15

### ISSUE THREAD

**Issue by hetzed** [philbertphotos/osticket-multildap-auth#49]
Hi,
I have installed
osTicket v1.15 (d5ee0df)
Apache/2.4.38 (Debian)
PHP 7.3.27-1~deb10u1

I had to bought adSync, but I really want to use this plugin because it lets upload different attributes from AD. 
Any clues why it doesn't work?

**Comment by SamehElsakka** [comment]
Hi ,i have the same issue the plugin is not working on version 1.15

**Comment by Malakii** [comment]
Also running into issues with this plugin on osTicket v1.15.3.

**Comment by philbertphotos** [comment]
I have this version installed and fixing it. Should be solved by weekend.

**Comment by broekel** [comment]
Hello right at the moment I have installed 
os Ticket Version: 1.15.3.1
Apache: 2.4.38
MySQL-Version: 10.3.29
PHP-Version: 7.3.29-1

I tested adSync 1.2.0 and adSync 1.2.2. Both versions are buggy.
Now I tried Multi-LDAP Authentification but it doesn’t work either. 
I need the right Search Filter parameter under „Useful only for Information lookups“.
You wrote in this post that you are also working on a sync-problem. Is it solved?
